### ISSUE TITLE
Headings Should Not Be Empty

### ISSUE THREAD

**Issue by christhepianist** [samvera/hyku#3049]
vpat.hykudemo.org
Version 7.1

### Descriptive Summary
In a SortSite testing, we found many pages that have a heading with no content. Here is the information we have:
- Description: Headings should not be empty
- Lines: We have 84 instances of this type of error that are on lines 396 and 400. 
- Guideline: https://www.w3.org/WAI/WCAG22/Techniques/general/G130
- Help: Add text to the heading, or alt text if the heading contains an image. Screen readers read out page headings, allowing users to quickly skip to a section, but some older screen readers do not ignore empty headings.
- URL: https://vpat.hykudemo.org/files/6980c798-dfb3-4394-87bb-72ff161c1fc7/stats?locale=de (one of many identified below)
- Reference: https://www.w3.org/TR/2012/NOTE-WCAG20-TECHS-20120103/G130

URLs identified with this error:
https://vpat.hykudemo.org/files/6980c798-dfb3-4394-87bb-72ff161c1fc7/stats?locale=de
https://vpat.hykudemo.org/files/6980c798-dfb3-4394-87bb-72ff161c1fc7/stats?locale=es
https://vpat.hykudemo.org/files/6980c798-dfb3-4394-87bb-72ff161c1fc7/stats?locale=fr
https://vpat.hykudemo.org/files/6980c798-dfb3-4394-87bb-72ff161c1fc7/stats?locale=it
https://vpat.hykudemo.org/files/6980c798-dfb3-4394-87bb-72ff161c1fc7/stats?locale=pt-BR
https://vpat.hykudemo.org/files/6980c798-dfb3-4394-87bb-72ff161c1fc7/stats?locale=zh
https://vpat.hykudemo.org/files/6980c798-dfb3-4394-87bb-72ff161c1fc7/stats?locale=en
https://vpat.hykudemo.org/files/b2e62b84-5490-4bb5-acd1-96ec6b64b5e3/stats?locale=de
https://vpat.hykudemo.org/files/b2e62b84-5490-4bb5-acd1-96ec6b64b5e3/stats?locale=es
https://vpat.hykudemo.org/files/b2e62b84-5490-4bb5-acd1-96ec6b64b5e3/stats?locale=fr
https://vpat.hykudemo.org/files/b2e62b84-5490-4bb5-acd1-96ec6b64b5e3/stats?locale=it
https://vpat.hykudemo.org/files/b2e62b84-5490-4bb5-acd1-96ec6b64b5e3/stats?locale=pt-BR
https://vpat.hykudemo.org/files/b2e62b84-5490-4bb5-acd1-96ec6b64b5e3/stats?locale=zh
https://vpat.hykudemo.org/files/b2e62b84-5490-4bb5-acd1-96ec6b64b5e3/stats?locale=en
https://vpat.hykudemo.org/files/ab6249c5-5170-456c-8368-9a72b4075747/stats?locale=de
https://vpat.hykudemo.org/files/ab6249c5-5170-456c-8368-9a72b4075747/stats?locale=es
https://vpat.hykudemo.org/files/ab6249c5-5170-456c-8368-9a72b4075747/stats?locale=fr
https://vpat.hykudemo.org/files/ab6249c5-5170-456c-8368-9a72b4075747/stats?locale=it
https://vpat.hykudemo.org/files/ab6249c5-5170-456c-8368-9a72b4075747/stats?locale=pt-BR
https://vpat.hykudemo.org/files/ab6249c5-5170-456c-8368-9a72b4075747/stats?locale=zh
https://vpat.hykudemo.org/files/ab6249c5-5170-456c-8368-9a72b4075747/stats?locale=en
https://vpat.hykudemo.org/files/740fa080-26dd-4cdc-96bd-ec7feee547a5/stats?locale=de
https://vpat.hykudemo.org/files/740fa080-26dd-4cdc-96bd-ec7feee547a5/stats?locale=es
https://vpat.hykudemo.org/files/740fa080-26dd-4cdc-96bd-ec7feee547a5/stats?locale=fr
https://vpat.hykudemo.org/files/740fa080-26dd-4cdc-96bd-ec7feee547a5/stats?locale=it
https://vpat.hykudemo.org/files/740fa080-26dd-4cdc-96bd-ec7feee547a5/stats?locale=pt-BR
https://vpat.hykudemo.org/files/740fa080-26dd-4cdc-96bd-ec7feee547a5/stats?locale=zh
https://vpat.hykudemo.org/files/740fa080-26dd-4cdc-96bd-ec7feee547a5/stats?locale=en
https://vpat.hykudemo.org/files/c9bd9ecf-1872-4b53-9675-a4496c5067f5/stats?locale=de
https://vpat.hykudemo.org/files/c9bd9ecf-1872-4b53-9675-a4496c5067f5/stats?locale=es
https://vpat.hykudemo.org/files/c9bd9ecf-1872-4b53-9675-a4496c5067f5/stats?locale=fr
https://vpat.hykudemo.org/files/c9bd9ecf-1872-4b53-9675-a4496c5067f5/stats?locale=it
https://vpat.hykudemo.org/files/c9bd9ecf-1872-4b53-9675-a4496c5067f5/stats?locale=pt-BR
https://vpat.hykudemo.org/files/c9bd9ecf-1872-4b53-9675-a4496c5067f5/stats?locale=zh
https://vpat.hykudemo.org/files/c9bd9ecf-1872-4b53-9675-a4496c5067f5/stats?locale=en
https://vpat.hykudemo.org/files/5584dabf-bb84-4b9e-a75d-3af4d6798ec9/stats?locale=de
https://vpat.hykudemo.org/files/5584dabf-bb84-4b9e-a75d-3af4d6798ec9/stats?locale=es
https://vpat.hykudemo.org/files/5584dabf-bb84-4b9e-a75d-3af4d6798ec9/stats?locale=fr
https://vpat.hykudemo.org/files/5584dabf-bb84-4b9e-a75d-3af4d6798ec9/stats?locale=it
https://vpat.hykudemo.org/files/5584dabf-bb84-4b9e-a75d-3af4d6798ec9/stats?locale=pt-BR
https://vpat.hykudemo.org/files/5584dabf-bb84-4b9e-a75d-3af4d6798ec9/stats?locale=zh
https://vpat.hykudemo.org/files/5584dabf-bb84-4b9e-a75d-3af4d6798ec9/stats?locale=en

### Steps to Reproduce the Error
* Run the SortSite test again

### Acceptance Criteria
* Run SortSite and not see this error

### Related Work
* Slack thread that discusses this issue with @maxkadel: https://samvera.slack.com/archives/C0F9JQJDQ/p1777924567637309
*
